maldoca/base/file.h

```diff
@@ -32,6 +32,8 @@
 #include "google/protobuf/message.h"
 #include "maldoca/base/statusor.h"
 
+using base::FilePath;
```

Comment:
also please use #include "mini_chromium/base/... " not "base/... "  I thought I had removed visibility of the bare base.